_layouts/people.html

```diff
@@ -274,132 +274,132 @@ <h2>Details</h2>
         {% if ols-1-people contains username %}
             {% assign roles = '' %}
             {% if ols-1-participants contains username %}
-                {% capture roles %}{{ roles }}, project lead{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 project lead{% endcapture %}
             {% endif %}
             {% if ols-1-mentors contains username %}
-                {% capture roles %}{{ roles }}, mentor{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 mentor{% endcapture %}
             {% endif %}
             {% if ols-1-experts contains username %}
-                {% capture roles %}{{ roles }}, expert{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 expert{% endcapture %}
             {% endif %}
             {% if ols-1-speakers contains username %}
-                {% capture roles %}{{ roles }}, speaker{% endcapture %}
+                {% capture roles %}{{ roles }},OLS-1 speaker{% endcapture %}
             {% endif %}
             {% if ols-1-hosts contains username %}
-                {% capture roles %}{{ roles }}, call host{% endcapture %}
```

Comment:
I think now it is not call hosts, but facilitators. Could you change that?  You can look at the table on the main  page to check

Reply:
Yes yes. That's correct.

Reply:
Done. 😀

Reply:
Can you also change that names are  checked on the list of facilitators and not hosts?

Reply:
Just to be sure what you mean...

I have to change every instance of `ols-1-hosts`  to `ols-1-facilitators`?

Reply:
It would mean also changing the loops above (starting on line 41) to extract not the hosts but the facilitators.
Is it more clear?

Reply:
Thank you! I'll get started.

Reply:
Does it look right, please? 🙏